python25.pxd

```diff
@@ -0,0 +1,629 @@
+# From: Eric Huss <e-huss@netmeridian.com>

```

Comment:
When compiling with the configuration in `comp.py`, the Cython build complains that it can't find python25 (`from python25 cimport ...` in _line_profiler.pyx). If I switch it to `from .python25 cimport ...`, it complains that a relative cimport beyond the main package isn't allowed. I was lazy and just copied the python25 file to the main directory, but in retrospect I probably could have fixed it by passing the line_profiler directory to `include_dirs` in `cythonize`.

Reply:
Fixed in https://github.com/pyutils/line_profiler/pull/165/commits/2017db68c888bbe5c0c619501d9a88c36e0a34b7